pom.xml

```diff
@@ -21,7 +21,25 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
-
+		<dependency>
+			<groupId>com.mysql</groupId>
+			<artifactId>mysql-connector-j</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>2.0</version>
+		</dependency>
+		<dependency>
```

Comment:
How about a testing framework like junit or junit5?

Reply:
The transitive version of a dependency might have known bugs or issues. I had to explicitly add this dependency because I got a warning/error in the **spring-boot-starter** dependency while the build ran in the GitHub action. So to avoid problems, I had to add the dependency and control the version.